### PR TITLE
Fixes part of #2834: changed lambda function argument name in WelcomeViewModelTest

### PR DESCRIPTION
**Pull request by ravi9607**
## Explanation
Changed the argument names of lambda functions in WelcomeViewModelTest from 'it' to something more descriptive.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [ ] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by prayutsu** [comment]
@ravi9607 Before making a PR to fix some issue, please make sure that you make a comment on that issue and wait until you get assigned to the issue(or part of the issue). This is necessary to avoid Multiple PRs from different contributors for the same issue.

File: app/src/sharedTest/java/org/oppia/android/app/home/WelcomeViewModelTest.kt [hunk 1]
```diff
@@ -150,16 +150,20 @@ class WelcomeViewModelTest {
...
-    ).use {
-      it.onActivity {
-        setUpTestFragment(it)
+    ).use { activityScenario ->
+      activityScenario.onActivity { homeFragmentTestActivity ->
+        setUpTestFragment(homeFragmentTestActivity)
         val welcomeViewModelProfile1Morning = createBasicWelcomeViewModel(testFragment)
         val welcomeViewModelProfile1MorningCopy = createBasicWelcomeViewModel(testFragment)
-        assertThat(welcomeViewModelProfile1Morning).isEqualTo(welcomeViewModelProfile1MorningCopy)
+        assertThat(welcomeViewModelProfile1Morning).isEqualTo(
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/WelcomeViewModelTest.kt:158]: why are we breaking these lines into multiple linees?

**Review by anandwana001 — CHANGES_REQUESTED** [review]

**Comment by anandwana001** [comment]
@ravi9607 Please perform the second point from the checklist in the PR description.

**Comment by ravi9607** [comment]
How to solve this error in pushing

Starting audit...
 Audit done.
Checkstyle lint check completed successfully
Protobuf lint check not available on msys
error: failed to push some refs to 'https://github.com/ravi9607/oppia-android.git'

**Comment by anandwana001** [comment]
@ravi9607 Seems like a bug we have, solving this in the PR https://github.com/oppia/oppia-android/pull/2849/files
Please wait till the above-mentioned PR get's merged. 

by the time I suggest to look for other issues from the list here - https://github.com/oppia/oppia-android/issues?q=is%3Aopen+is%3Aissue+milestone%3ABeta+no%3Aassignee+-label%3A%22good+first+issue%22+NOT+a11y+NOT+blocked

**Comment by anandwana001** [comment]
@ravi9607 It looks like you need to create a new PR for this, as this PR is from the develop branch. You have to create a new branch and then make a PR. 
https://github.com/oppia/oppia-android/wiki/Android-Studio-UI-based-Github-workflow

Closing this PR.

**Comment by ravi9607** [comment]
same error with a new branch

**Pull request closed without merging**